.travis.yml

```diff
@@ -80,6 +80,20 @@ matrix:
         - npm run build -- --scope "@loopback/test-repository-postgresql" --include-dependencies
         - psql -U postgres -a -f acceptance/repository-postgresql/test/schema.sql
         - cd acceptance/repository-postgresql && npm run mocha
+    - node_js: '10'
+      os: linux
+      env:
+        - TASK=test-repository-cloudant
+      install:
+        # - rm -rf node_modules && yarn cache clean
+        - npm ci && npm run docker:setup && source cloudant-config.sh && pwd
+
+      script:
+        - source cloudant-config.sh
```

Comment:
This line alone does nothing as far as I can tell since each `script` command should be run in a new shell so the env vars exported by `cloudant-config.sh` just go away after this line.